books_to_scrape/spiders/books.py

```diff
@@ -0,0 +1,53 @@
+import re
+from pathlib import Path
+from types import new_class
```

Comment:
The import `from types import new_class` is not used in the code and can be removed to clean up the imports.